client/src/pages/mainPage/components/home/Home.js

```diff
@@ -156,8 +208,117 @@ export function Home() {
+        </div>
+
+        <h4 className="font-weight-bold dark-grey-text mt-4">Anna Deynah</h4>
+        <h6 className="font-weight-bold blue-text my-3">Web Designer</h6>
+        <p className="font-weight-normal dark-grey-text">
+          <i className="fas fa-quote-left pr-2"></i>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quod
+          eos id officiis hic tenetur quae quaerat ad velit ab hic tenetur.</p>
+
+        <div className="orange-text">
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star-half-alt"> </i>
+        </div>
+      </div>
+
+    </div>
+
+    <div className="col-md-4 mb-md-0 mb-5">
+
+      <div className="testimonial">
+
+        <div className="avatar mx-auto">
+          <img src="https://mdbootstrap.com/img/Photos/Avatars/img%20(8).webp" className="rounded-circle z-depth-1 img-fluid" alt="ph" />
+        </div>
+
+        <h4 className="font-weight-bold dark-grey-text mt-4">John Doe</h4>
+        <h6 className="font-weight-bold blue-text my-3">Web Developer</h6>
+        <p className="font-weight-normal dark-grey-text">
+          <i className="fas fa-quote-left pr-2"></i>Ut enim ad minima veniam, quis nostrum exercitationem ullam
+          corporis suscipit laboriosam, nisi ut aliquid commodi.</p>
+
+        <div className="orange-text">
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+        </div>
+      </div>
+
+    </div>
+
+    <div className="col-md-4">
+
+      <div className="testimonial">
+
+        <div className="avatar mx-auto">
+          <img src="https://mdbootstrap.com/img/Photos/Avatars/img%20(10).webp" className="rounded-circle z-depth-1 img-fluid" alt="ph2" />
+        </div>
+
+        <h4 className="font-weight-bold dark-grey-text mt-4">Maria Kate</h4>
+        <h6 className="font-weight-bold blue-text my-3">Photographer</h6>
+        <p className="font-weight-normal dark-grey-text">
+          <i className="fas fa-quote-left pr-2"></i>At vero eos et accusamus et iusto odio dignissimos ducimus qui
+          blanditiis praesentium voluptatum deleniti atque corrupti.</p>
+
+        <div className="orange-text">
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="fas fa-star"> </i>
+          <i className="far fa-star"> </i>
+        </div>
+      </div>
+
+    </div>
+
+
+  </div>
+
+
+</section>
+<section>
+<div className="text-center pt-1 mb-5 pb-1">
+	<h3 className="bg-primary btn-block fa-lg gradient-custom-2 mb-3">Effortless Energiser</h3>
+
+	<p>Do the things you do,but better!</p>
+	</div>
+</section>
+{/* Adem */}
```

Comment:
@ademalagoz clean code you have written there.
But please try not to leave these comments in the code. We really don't want to push them to the main branch.